docs/UserGuide.adoc

```diff
@@ -253,52 +254,65 @@ NOTE: Assume entry at index 1 is `[-] Laksa ($10.50) Date: 18/10/2019 Remarks: U
 
 [[UndoTag]]
 === Undoing the previous undoable command: `undo`
+Do not panic if you have accidentally made a mistake in THRIFT, like deleting the wrong transaction.
+THRIFT provides an `undo` command to save your trouble! You can revert to the previous state
+of THRIFT using `undo`.
 
 Format: `undo`
 
-NOTE: Undoable commands are commands that modify the data of THRIFT application.
+NOTE: `Undo` can only apply to undoable commands. Undoable commands refers to commands that modify the data of the THRIFT application.
 
-.List of undoable commands[[Undoable]]:
-* <<ExpenseTag, add_expense>>
-* <<IncomeTag, add_income>>
-* <<DeleteTag, delete>>
-* <<UpdateTag, update>>
-* <<CloneTag, clone>>
-* <<TagTag, tag>>
-* <<UntagTag, untag>>
-
-Examples:
+WARNING: You cannot undo undoable commands once the application exits.
 
-* `add_expense n/Laksa v/10.50 t/Lunch t/Food` +
-  `undo`
+.List of undoable commands[[Undoable]]:
+* `<<ExpenseTag, add_expense>>`
+* `<<IncomeTag, add_income>>`
+* `<<DeleteTag, delete>>`
+* `<<UpdateTag, update>>`
+* `<<CloneTag, clone>>`
+* `<<TagTag, tag>>`
+* `<<UntagTag, untag>>`
+* `<<BudgetTag, budget>>`
 
-* `add_income n/Allowance v/1000 t/Monthly` +
-  `list` +
-  `undo`
+Examples: +
+Suppose you want to delete the second transaction (Bursary) in THRIFT, you remove the third transaction instead!
```

Comment:
Would a "but" in between sentences make them flow better? e.g. "Suppose you want to delete the second transaction (Bursary) in THRIFT, **but** you remove the third transaction instead!"